jekyll/_cci2/bitbucket-integration.adoc

```diff
@@ -199,12 +199,12 @@ You can add the key to known_hosts by running the following command:
 ssh-keyscan bitbucket.com >> ~/.ssh/known_hosts
 ```
 
-## Rename organizations and repositories
-{: #rename-organizations-and-repositories }
+[#rename-organizations-and-repositories]
+== Rename organizations and repositories
 
-If you would like to rename your organization or repository, please follow the [Rename organizations and repositories]({{site.baseurl}}/rename-organizations-and-repositories) guide to make sure you do not lose access to environment variables or contexts in the process.
+If you would like to rename your organization or repository, please follow the <<rename-organizations-and-repositories,Rename organizations and repositories>> guide to make sure you do not lose access to environment variables or contexts in the process.
```

Comment:
```suggestion
If you would like to rename your organization or repository, please follow the xref:rename-organizations-and-repositories[Rename organizations and repositories] guide to make sure you do not lose access to environment variables or contexts in the process.
```

Reply:
the way this was before it was referencing itself, if you see what I mean

Reply:
Yes but this is actually referencing a section that now exists on this page, which I think we should keep in carrots to distinguish it as an internal page section link rather than a docs link